pom.xml

```diff
@@ -211,11 +211,6 @@
     <dependency>
```

Comment:
I am also thinking that this should be removed from here and just let the transitive dependency propagate though at all the correct scopes.

Reply:
IIUC it is a `provided` dependency of core, so it would not be available as a transitive dep.